app/main.py

```diff
@@ -1,6 +1,17 @@
-from typing import Callable
+from typing import Callable, Any
+
+
+results = {}
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    def inner(*args: Any) -> Any:
+        global results
+        if (func, args) in results:
```

Comment:
```suggestion
        if (func.__name__, args) in results:
```
I think better to check func name

Reply:
I dont know why but in such case even when i am using _functools_ I have one test failed.

![Знімок екрана 2024-03-28 102827](https://github.com/mate-academy/py-cache-decorator/assets/137896454/dff625bb-314b-4fa6-9fc5-e14a1c04d01a)